modules/core/src/test/java/org/apache/ignite/util/GridCommandHandlerClusterByClassTest.java

```diff
@@ -364,10 +368,52 @@ public void testHelp() {
         for (CommandList cmd : CommandList.values())
             assertContains(log, testOutStr, cmd.toString());
 
-        assertContains(log, testOutStr, "Control utility script");
-        assertContains(log, testOutStr, CommandHandler.UTILITY_NAME);
-
         assertNotContains(log, testOutStr, "Control.sh");
+
+        checkHelp(testOutStr, "org.apache.ignite.util/control.sh_help.output");
+    }
+
+    /**
+     * Checks that golden copy of output and current output are same.
+     *
+     * @param output Current output.
+     * @param resourceName Name of resource with golden copy on output.
+     * @throws Exception If something goes wrong.
+     */
+    private void checkHelp(String output, String resourceName) throws Exception {
+        URL goldenCopyResource = getClass().getClassLoader().getResource(resourceName);
+
+        assertNotNull("Golden copy not found. path: " + resourceName, goldenCopyResource);
```

Comment:
It seems this check is not really necessary, you can throw exception from readResource(). as getResourceAsStream returns null if resource is not found.